lib/push.js

```diff
@@ -264,6 +275,25 @@ module.exports = function (log, db, config) {
       return this.pushToAllDevices(uid, 'deviceDisconnected', options)
     },
 
+
+    /**
+     * Notifies connected devices about a signin confirmation request.
+     *
+    */
+
+    notifyVerifyLogin: function notifyVerifyLogin(uid, devices) {
+      var data = Buffer.from(JSON.stringify({
+        command: PUSH_COMMANDS.VERIFY_LOGIN_NOTIFICATION,
+        data: {
+          title: 'Sign-in Request',
+          body: '',
```

Comment:
Right, so we'd want to make use of `gettext` to be able to provide translated notification text. This is the text shown in the notification popup itself. We could either add in `gettext` and the translator directly to this file, or a more hefty change could be to add a `push` file in `senders`... I'm not sure yet which would be the better direction..